### ISSUE TITLE
Ingest datasets without storing actual niftis

### ISSUE THREAD

**Issue by tyarkoni** [neuroscout/neuroscout#342]
I don't think we need the nifti volumes for anything other than detecting the run length, which is in the header. Let's see if we can systematiclaly truncate the image files to keep just the header, which will allow us to conserve space when moving to production. Failing that, we can always just copy the run length into the .json sidecars, but it's conceivable we might want the headers for other reasons, so let's try to keep them. @effigies, did you write some code to do this when creating the `synthetic/` BIDS example dataset?

**Comment by effigies** [comment]
I think I actually used `rsync`, `gunzip`, `truncate -s 352`, and `gzip`.

**Comment by effigies** [comment]
Doing this with nibabel would be slightly tricky, though it should get easier once the `to_bytes` interface is merged (nipy/nibabel#644).

Also, just to cross reference the synthetic dataset: INCF/BIDS-examples#48

**Comment by adelavega** [comment]
Awesome, it looks like the commands @effigies provided do the trick. pybids (v0.8) can ingest everything properly as the headers as left untouched. 

To ingest datasets then do the following:
    for f in  */func/*.nii.gz; do gunzip $f && truncate -s 352 ${f:0:-3} && gzip ${f:0:-3}; done
    for f in  */anat/*.nii.gz; do gunzip $f && truncate -s 352 ${f:0:-3} && gzip ${f:0:-3}; done

Then in `derivatives`, get the processed dataset and place in `fmriprep`. Since those are datalad datasets, and we don't ingest using the preprocessed files, simply dont `GET` the niftis.

The constant that *does* have to downloaded (but could be removed later) is the stimuli. So that could end up being 200MB - 1GB per dataset. In theory, those could also be deleted once they are ingested as images (However, those will also take up space).

So my estimate is that it will be good to allocate 1-2GB per datasets on the production server.

**Comment by adelavega** [comment]
Closing this in favor of #600
